app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -308,6 +308,9 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="list_in_recent_files">最近文件列表</string>
     <string name="reload">刷新</string>
     <string name="wrap_words">折行</string>
+    <string name="show_line_numbers">显示行号</string>
```

Comment:
I see, what should I do, do I put them to string-not_translatable.xml?

Reply:
generally dont add translations (as in i.e. chinese) - only english. And here, if it is not translatable (like Markor or Markdown is not) put it in the not_translatable file. And otherwise in the strings.xml.

Translations will come then later in after merge.